src/main/java/ua/petproject/categoryservice/CategoryServiceIMPL.java

```diff
@@ -0,0 +1,36 @@
+package ua.petproject.categoryservice;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ua.petproject.model.Element;
+import ua.petproject.repository.CategoryDAO;
+
+@Service
+public class CategoryServiceIMPL implements CategoryService {
```

Comment:
```suggestion
public class CategoryServiceImpl implements CategoryService {
```
Rename class following the naming convention